Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
```

Comment:
It seems Google have their internal git system, and publish stable code to
the master branch for community, I will submit a issue to them later.

Mike McQuaid <notifications@github.com>于2016年5月18日 周三19:53写道：

> In Formula/breakpad.rb
> <https://github.com/Homebrew/homebrew-core/pull/1256#discussion_r63689084>
> :
>
> > @@ -0,0 +1,69 @@
> > +class Breakpad < Formula
> > +  desc "Components implement a crash-reporting system."
> > +  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
> > +
> > +  url "https://chromium.googlesource.com/breakpad/breakpad",
> > +      :using => :git,
> > +      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
>
> It'd be good to at least ask. Does Google have a "stable" branch or
> similar? The concern is if upstream are happy with us packaging a random
> commit.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/1256/files/c2c2e54da6540b2fd74176bf865026191a24f67b#r63689084>
>